proto/sound_sensor.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+
+// Trait for devices that measure sound/noise levels.
+service SoundSensorApi {
+  rpc GetSoundLevel(GetSoundLevelRequest) returns (SoundLevel) {}
+  rpc PullSoundLevel(PullSoundLevelRequest) returns (stream PullSoundLevelResponse) {}
+}
+
+service SoundSensorInfo {
+  rpc DescribeSoundLevel(DescribeSoundLevelRequest) returns (SoundLevelSupport) {}
+}
+
+// SoundLevel represents a sound/noise level.
+message SoundLevel {
+  // The measured sound/noise level, usually in dBA or dB. Use Info service to check.
+  // Optional. 0 means no sound or possibly broken sensor. Omitted means unknown.
```

Comment:
0 isn't a good choice to represent a broken sensor or no sound IMO.
0 is a reading you could in theory obtain from a working sound sensor in a very quiet space.
No sound would be whatever the noise floor of the sensor is. Or '-Infinity' I suppose.
Broken sensor should also result in an unset field if we can detect that it's broken.